homeassistant/components/recorder/__init__.py

```diff
@@ -383,6 +384,8 @@ def async_purge(now):
 
             try:
                 dbevent = Events.from_event(event)
+                if event.event_type == EVENT_STATE_CHANGED:
+                    dbevent.event_data = "{}"
```

Comment:
I don't know if this will break anything. We should definitely mention this on <https://data.home-assistant.io/docs/events#database-table>

Reply:
I looked for any other place with use the Events model and it should be fine (at least with the core codebase).  Definitely needs to be documented though

Reply:
https://www.home-assistant.io/docs/backend/database/#schema also needs to be updated

Reply:
https://data.home-assistant.io/docs/states needs to be updated as well

Reply:
https://github.com/home-assistant/home-assistant.io/pull/13784 and https://github.com/home-assistant/data.home-assistant/pull/63 opened